Resources/Textures/_RMC14/Interface/health_hud_holocard.rsi/meta.json

```diff
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Created by blueDev2 [https://github.com/blueDev2]. Attribution may be done by linking to the github profile",
+  "copyright": "Created by Vermidia",
```

Comment:
If you want to, consider licensing these under CC-BY-SA-4.0 or CC-BY-4.0, it makes licensing easier for us